extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/spark/kyuubi/lineage/LineageConf.scala

```diff
@@ -28,4 +30,12 @@ object LineageConf {
       .booleanConf
       .createWithDefault(false)
 
+  val DISPATCHERS = ConfigBuilder("spark.kyuubi.plugin.lineage.dispatchers")
```

Comment:
please add .doc and .version

Reply:
better to mention this config in lineage docs.

Reply:
> better to mention this config in lineage docs.

Updated `docs/extensions/engines/spark/lineage.md`